Client/src/Views/Records/RecordUI.jsx

```diff
@@ -74,8 +74,10 @@ class RecordUI extends Component {
 
   _scrollToActiveSection() {
     this.unmonitorActiveSection();
-    let domNode = document.getElementById(location.hash.slice(1));
+    let sectionId = location.hash.slice(1);
+    let domNode = document.getElementById(sectionId);
     if (domNode != null) {
+      this.props.updateSectionVisibility(sectionId, true);
```

Comment:
I can imagine that the location hash could be attached to an element that is a child of a section, such as a table row. Should we find the closest ancestor section id?

Reply:
Just committed something for this!